internal/controller/streams.go

```diff
@@ -968,13 +942,18 @@ func ConnectStreams(ctx context.Context, logger *zap.Logger, envMap EnvironmentC
 				failureReason = "Configuration update stream closed"
 			case <-ovnkDone:
 				failureReason = "OVN-K network flow stream closed"
+			case <-flowCacheOutReaderDone:
+				failureReason = "Flow cache reader failed"
+			case <-flowCacheRunDone:
+				failureReason = "Flow cache running process failed"
 			}
 			authConContextCancel()
 		}
 		logger.Warn("One or more streams have been closed; closing and reopening the connection to CloudSecure",
 			zap.String("failureReason", failureReason),
 			zap.Int("attempt", attempt),
 		)
+		resetTimer.Reset(jitterTime(5*time.Second, 0.20))
```

Comment:
Wow, this scares me.
Why is this necessary?
It seems like it is necessary, but how come we've missed that until now???

In case the connection fails, we already reset this time to 10s-jitter here:
https://github.com/illumio/cloud-operator/pull/252/files#diff-65d4066e2022c168a97a2083474d348fa6d5d1fffb6e8d7378282935ad187940R829
Why reset it afterwards to 5s?

I would rather move this after the `if err != nil` block here, maybe in an `else` clause to make it more clear:
https://github.com/illumio/cloud-operator/pull/252/files#diff-65d4066e2022c168a97a2083474d348fa6d5d1fffb6e8d7378282935ad187940R833
So that both `Reset` calls are close to each other.